src/reducers/RecipeReducer.js

```diff
@@ -40,6 +41,33 @@ const RecipeReducer = createReducer(initialState, {
     isFetching: false,
     recipeRecs,
   }),
+  [actionTypes.FETCH_LIKED_RECIPES]: (state) => ({
+    ...state,
+    isFetching: true,
+  }),
+  [actionTypes.FETCH_RECIPE_TO_LIKE]: (state) => ({
+    ...state,
+    isFetching: true,
+  }),
+  [actionTypes.FETCH_RECIPE_TO_UNLIKE]: (state) => ({
+    ...state,
+    isFetching: true,
+  }),
+  [actionTypes.RECEIVE_LIKED_RECIPES]: (state, { likedRecipes }) => ({
+    ...state,
+    isFetching: false,
+    likedRecipes,
+  }),
+  [actionTypes.RECEIVE_RECIPE_TO_LIKE]: (state, { recipeDetails }) => ({
+    ...state,
+    isFetching: false,
+    recipeDetails,
+  }),
+  [actionTypes.RECEIVE_RECIPE_TO_UNLIKE]: (state, { recipeDetails }) => ({
+    ...state,
+    isFetching: false,
+    recipeDetails,
+  }),
```

Comment:
we're not necessarily re-fetching the recipe after liking, make this also redundant

Reply:
so you don't want it in the reducer at all?

Reply:
@seanyles

Reply:
It doesn't hurt. I'm fine with it